pkg/util/indexparams/index_params.go

```diff
@@ -299,6 +299,14 @@ func SetDiskIndexBuildParams(indexParams map[string]string, fieldDataSize int64)
 	return nil
 }
 
+func SetBitmapIndexLoadParams(params *paramtable.ComponentParam, indexParams map[string]string) {
+	_, exist := indexParams[common.IndexOffsetCacheEnabledKey]
+	if exist {
```

Comment:
just set once ??

Reply:
if index param has this key, means user set it, use user param.
if index param not have this key, using milvus.yaml configuration.